.travis.yml

```diff
@@ -4,7 +4,7 @@ addons:
   apt:
     packages:
     - zsh
-bundler_args: --without production development --deployment --jobs=3 --retry=3
```

Comment:
Pls move gems to the right place instead of installing all production gems on travis.